src/app/StudentView.tsx

```diff
@@ -45,7 +45,10 @@ function renderGrade(grade: { course: string, grade: number }) {
         iconColor = "red.500";
     }
     return <ListItem key={grade.course + "." + grade.grade}>
-        <ListIcon as={icon} color={iconColor}/>{grade.course}: {grade.grade}%</ListItem>;
+        <ListIcon as={icon} color={iconColor}/>{grade.course}: {grade.grade}%
+        <Button>Open</Button>
+        </ListItem>;
+        
```

Comment:
Front end not finished.

Reply:
Yes, in the given timeframe, we implemented the backend and the frontend has only a button which currently does not call the backend. This can be updated.

Reply:
Yeah, that would be good.